src/angular-app/bellows/apps/changepassword/change-password-app.component.html

```diff
@@ -1,39 +1,76 @@
 <script src="/dist/zxcvbn/zxcvbn.js"></script>
 <div class="content container" data-ng-cloak>
-    <div class="row justify-content-center">
+  <div class="row justify-content-center">
     <div class="col-12 col-lg-8" id="form-container">
-        <sil-notices></sil-notices>
-        <form data-ng-submit="$ctrl.updatePassword()" name="passwordForm">
-            <br />
-            <div class="form-group row">
-                <label class="col-form-label col-md-4 text-md-right">Password</label>
-                <div class="col-md-8">
-                    <input required class="form-control" id="change-password-input" type="password" data-ng-model="$ctrl.password" name="password"
-                           pattern=".{7,}" title="Passwords must be at least 7 characters" data-pui-auto-focus="true">
-                </div>
-            </div>
-            <div class="form-group row">
-                <label class="col-form-label col-md-4 text-md-right">Confirm password</label>
-                <div class="col-md-8">
-                    <input required class="form-control" id="change-password-confirm-input" type="password" data-ng-model="$ctrl.confirm_password"
-                           data-pui-validate=" '$value===$ctrl.password' " name="confirm_password"
-                           pattern=".{7,}" title="Passwords must be at least 7 characters">
-                </div>
-                <span data-ng-show='$ctrl.password && passwordForm.confirm_password.$error.validator'>
-                    <i class="fa fa-exclamation-circle" title="Passwords do not match" style="position: absolute; padding-left: 5px; padding-top: 8px;"></i>
-                </span>
-                <span data-ng-show='$ctrl.password && $ctrl.password === $ctrl.confirm_password'>
-                    <i id="change-password-match" class="fa fa-check" title="Password OK" style="position: absolute; padding-left: 5px; padding-top: 8px;"></i>
-                </span>
-            </div>
-            <div class="form-group row">
-                <div class="col-md-8 offset-md-4">
-                    <button type="submit" class="btn btn-primary" id="change-password-submit-button"
-                            data-ng-disabled="$ctrl.password !== $ctrl.confirm_password || passwordForm.$invalid">
-                        Change Password</button>
-                </div>
-            </div>
-        </form>
-    </div>
+      <sil-notices></sil-notices>
+      <form data-ng-submit="$ctrl.updatePassword()" name="passwordForm">
+        <br />
+        <div class="form-group row">
+          <label class="col-form-label col-md-4 text-md-right">Password</label>
+          <div class="col-md-8">
+            <input
```

Comment:
I guess we can discuss these types of changes more, I don't prefer these vertical layouts like this

Reply:
This is something I had to get used to, but ultimately it's easier to read isn't it?

Reply:
depends on what you're looking for.  I'm typically most interested in the html structure primarily and the args secondarily so it's for me to see the sibling nature of elements when everything is on one line.